app/settings/tokens/edit/route.ts

```diff
@@ -11,7 +11,7 @@ export default class SettingsTokensEditRoute extends Route.extend({
         try {
             return yield this.store.findRecord('token', id, { reload: false });
         } catch (e) {
-            this.replaceWith('not-found', this.router.currentURL.slice(1));
+            this.replaceWith('not-found', this.router.currentURL ? this.router.currentURL.slice(1) : '');
```

Comment:
I'm curious when there's no `currentURL`?

Reply:
Here, it should always be set, but it is `null` during integration tests. I added this because was planning to change the type from `string` to `string | null` (was planning to PR DT). [This](https://github.com/CenterForOpenScience/ember-osf-web/pull/399/files#diff-6e957e5840475a5ef0730341ca127ddbR55) is where I've used `currentURL` it in a component. Now that I think about it, though, for integration tests I could stub the router service and set `currentURL` to empty string.